src/components/query-viewer/controller/query-viewer-controller.tsx

```diff
@@ -182,6 +176,26 @@ export class QueryViewerController {
    */
   @Prop() readonly metadataName: string;
 
+  /**
+   * This is the GxQuery base URL. It will required when property useGxQuery = true
```

Comment:
The `baseUrl` of the query-viewer-controller is used in both GeneXus' and GXQuery applications.